packages/playground/src/weblets/tf_gitea.vue

```diff
@@ -171,29 +187,47 @@ async function deploy() {
       ],
     });
 
-    layout.value.reloadDeploymentsList();
-    layout.value.setStatus("success", "Successfully deployed a Gitea machine.");
-    layout.value.openDialog(vm, deploymentListEnvironments.gitea);
+    if (!selectionDetails.value?.domain?.enableSelectedDomain) {
+      vm[0].customDomain = selectionDetails.value?.domain?.customDomain;
+      finalize(vm);
+      return;
+    }
+
+    try {
+      layout.value.setStatus("deploy", "Preparing to deploy gateway...");
+      await deployGatewayName(grid, selectionDetails.value.domain, {
+        subdomain,
+        ip: vm[0].interfaces[0].ip,
+        port: 3000,
```

Comment:
I think the port here needs to be added as a constant at the top of the file. I think it can be changed later in the Dockerfile, so setting it at the top of the file will be clearer. What do you think?

Reply:
The thing is, this is how it's implemented in all solutions. If it will be changed, we can open an issue to edit it in all solutions at once.